pkg/controllers/state/cluster.go

```diff
@@ -47,20 +46,17 @@ type observerFunc func(string)
 
 // Cluster maintains cluster state that is often needed but expensive to compute.
 type Cluster struct {
-	kubeClient    client.Client
-	cloudProvider cloudprovider.CloudProvider
-	clock         clock.Clock
-
-	// Pod Specific Tracking
-	antiAffinityPods sync.Map // mapping of pod namespaced name to *v1.Pod of pods that have required anti affinities
-
-	nominatedNodes         *cache.Cache
+	kubeClient             client.Client
+	cloudProvider          cloudprovider.CloudProvider
+	clock                  clock.Clock
 	nominatedNodeObservers atomicutils.Slice[observerFunc]
 
-	// Node Status & Pod -> Node Binding
-	mu       sync.RWMutex
-	nodes    map[string]*Node                // node name -> node
-	bindings map[types.NamespacedName]string // pod namespaced named -> node name
+	// State: Node Status & Pod -> Node Binding
+	mu               sync.RWMutex
```

Comment:
The Go standard library uses [mutex hats](https://dmitri.shuralyov.com/idiomatic-go#mutex-hat) somewhat consistently.  It's just a quick way to know which data members are protected by which mutex if any.  In this case, nominatedNodes and antiAffinityPods have their own build in locks and don't need the mutex.